low-level/sspl_ll_d

```diff
@@ -713,6 +716,26 @@ def send_thread_controller_request(state):
 #     logger.info(f"Received SIGUSR1 signal to set log level to '{log_level}'")
 #     logger.setLevel(log_level)
 
+def initialize_iem():
+    try:
+        EventMessage.init(component='sspl', source='S')
+        logger.info("IEM framework initiated successfully!!!")
+    except EventMessageError as e:
+        # create IEM_INIT_FAILED file and log current sspl pid.
+        create_file(IEM_INIT_FAILED)
+        with open(IEM_INIT_FAILED, 'w') as f:
+            f.write(str(os.getpid()))
+        logger.critical("Failed to initialize IEM Framework."
+                     f"ERROR:{e}")
+
+def create_file(path):
```

Comment:
such functions should be collected in common utility class, right now keep it under Utility class only. Ideally they need to be under specific classes, though that classification we will do separately

Reply:
done